lib/internal/Magento/Framework/App/Config/ScopeConfigInterface.php

```diff
@@ -17,14 +15,14 @@ interface ScopeConfigInterface
     /**
      * Default scope type
      */
-    const SCOPE_TYPE_DEFAULT = 'default';
+    public const SCOPE_TYPE_DEFAULT = 'default';
 
     /**
      * Retrieve config value by path and scope.
      *
      * @param string $path The path through the tree of configuration values, e.g., 'general/store_information/name'
      * @param string $scopeType The scope to use to determine config value, e.g., 'store' or 'default'
-     * @param null|int|string $scopeCode
+     * @param null|int|string|\Magento\Store\Api\Data\StoreInterface $scopeCode
```

Comment:
It could be `\Magento\Framework\App\ScopeInterface` instead of `\Magento\Store\Api\Data\StoreInterface`

Reply:
Perfect, thanks for your feedback :rocket: